client/components/chart/index.js

```diff
@@ -0,0 +1,213 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import classNames from 'classnames';
+import { Component, Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import D3Chart from './charts';
+import dummyOrders from './test/fixtures/dummy';
+import Legend from './legend';
+
+class WidgetCharts extends Component {
+	constructor() {
+		super( ...arguments );
+		this.getOrderedKeys = this.getOrderedKeys.bind( this );
+		this.handleLegendHover = this.handleLegendHover.bind( this );
+		this.handleLegendToggle = this.handleLegendToggle.bind( this );
+		this.updateDimensions = this.updateDimensions.bind( this );
+		this.handleChange = this.handleChange.bind( this );
+		this.getSomeOrders = this.getSomeOrders.bind( this );
+		const products = [
+			{
+				key: 'date',
+				selected: true,
+			},
+			{
+				key: 'Cap',
+				selected: true,
+			},
+			{
+				key: 'T-Shirt',
+				selected: true,
+			},
+			{
+				key: 'Sunglasses',
+				selected: true,
+			},
+			{
+				key: 'Polo',
+				selected: true,
+			},
+			{
+				key: 'Hoodie',
+				selected: true,
+			},
+		];
+		const someOrders = this.getSomeOrders( products );
```

Comment:
We should step away from names involving `orders` and `products`

Reply:
These are just for the dummy checkboxes and dummy data for testing. I'll ensure we steer clear of that wording once this is all hooked up.